src/apps/vpn/mozillavpn.cpp

```diff
@@ -2134,6 +2134,35 @@ void MozillaVPN::registerInspectorCommands() {
         MozillaVPN::instance()->controller()->quit();
         return QJsonObject();
       });
+
+  InspectorHandler::registerCommand(
+      "force_connection_health",
+      "Force VPN connection health stability. Possible values are: stable, "
+      "unstable, nosignal",
+      1, [](InspectorHandler*, const QList<QByteArray>& arguments) {
+        QJsonObject obj;
+        auto stabilityMode = arguments[1];
+
+        if (stabilityMode == "stable") {
+          MozillaVPN::instance()
+              ->connectionHealth()
+              ->overwriteStabilityForInspector(
+                  ConnectionHealth::ConnectionStability::Stable);
+        } else if (stabilityMode == "unstable") {
+          MozillaVPN::instance()
+              ->connectionHealth()
+              ->overwriteStabilityForInspector(
+                  ConnectionHealth::ConnectionStability::Unstable);
+        } else if (stabilityMode == "nosignal") {
+          MozillaVPN::instance()
+              ->connectionHealth()
+              ->overwriteStabilityForInspector(
+                  ConnectionHealth::ConnectionStability::NoSignal);
+        } else {
```

Comment:
Nit: should we have something to "reset" the forced value?

Reply:
That's what "stable" does, are you thinking of a different way of doing this? I could add another accepted argument "reset" but under the hood it would do the same thing as setting it back to stable.

Reply:
I meant like m_stabilityOverwritten=false so that i get "real" stablitty values on the next check